daemon/graphdriver/lcow/lcow.go

```diff
@@ -181,17 +181,17 @@ func InitDriver(dataRoot string, options []string, _, _ []idtools.IDMap) (graphd
 	}
 
 	// Make sure the dataRoot directory is created
-	if err := idtools.MkdirAllAs(dataRoot, 0700, 0, 0); err != nil {
+	if err := idtools.MkdirAllAndChown(dataRoot, 0700, idtools.IDPair{UID: 0, GID: 0}); err != nil {
```

Comment:
small nitpick (optional): declaring and assigning `zeroIDPair = idtools.IDPair{UID: 0, GID: 0}` and passing it in all the `MkdirAllAndChown`.

Reply:
I initially started doing that, but decided that it was only used in three locations here, and found this more readable (without having to look what `zeroIDPair` actually was).

For the other locations; I was thinking of adding a `idtools.GetRootIDPair(d.uidMaps, d.gidMaps)` utility, but kept that for a possible follow-up